XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java

```diff
@@ -101,7 +101,8 @@ public void delete(MembershipPayment model) {
     @Override
     public void insert(MembershipPayment model) throws RepositoryException {
         try {
-            sqlService.insert("payments", new Object[]{
+            String columns = ("MEM_ID, TYPE_OF_PAYMENT, AMOUNT, DATE, TIME");
+            sqlService.insert("PAYMENTS", new Object[]{
```

Comment:
Columns still aren't being passed to the update here.

Reply:
Oops!

Reply:
Sorted.